src/escpos/printer/file.py

```diff
@@ -73,6 +72,8 @@ def _raw(self, msg):
 
     def close(self):
         """Close system file."""
-        if self.device is not None:
-            self.device.flush()
-            self.device.close()
+        if not self._device:
+            return
+        self.device.flush()
+        self.device.close()
+        self.device = None
```

Comment:
I agree it makes more sense to restore the connection to False (Not initialized) after `close()`.
As far as I remember, the only reason to set `device` to None on `close()` was because we were checking `if self.device is None` in some places but I changed those checks to `if not self.device` so either False or None will pass the condition.
I missed that. Thanks!